### ISSUE TITLE
Trackpad scrolling left or right with two fingers = forward / back (MACOS)

### ISSUE THREAD

**Issue by friendlytherapist** [FreeTubeApp/FreeTube#1976]
**Is your feature request related to a usage problem (not a bug)? Please describe.**
Trackpad scrolling left or right with two fingers does not go forward/back a page. This feature is in all browsers on macos and id love to see it implemented.

**Describe the solution you'd like to see implemented**
Trackpad scrolling left or right with two fingers goes forward/back a page.

**Describe alternatives you've considered**
Not that I am aware of

**Comment by lothar-cell** [comment]
Yes, it would enhance the user experience greatly, please implement gestures!

**Comment by lothar-cell** [comment]
I comment on this, so it won't be closed automatically in 7 days.
